ark/phenotyping/som_utils.py

```diff
@@ -338,6 +443,12 @@ def compute_cell_cluster_counts(fovs, consensus_path,
     # NaN means the cluster wasn't found in the specified fov-cell pair
     cell_table = cell_table.fillna(0)
 
+    # remove all columns that are zero (should rarely happen)
```

Comment:
Why completely remove zero columns?

Reply:
@ngreenwald I discussed with her and this should never happen, only here in case the user somehow passes in a broken dataset. I'll remove this line.